### ISSUE TITLE
Changing disk_volume_size does not register as a change

### ISSUE THREAD

**Issue by Dombo** [terraform-providers/terraform-provider-aws#939]
Raising an issue as per the suggestion in https://github.com/hashicorp/terraform/pull/11619#issuecomment-310235056

### Terraform Version
v0.9.5

### Affected Resource(s)
Please list the resources as a list, for example:
- aws_launch_configuration

If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.

### Terraform Configuration Files
```hcl
resource "aws_launch_configuration" "bluegreen_launchconfig" {
  image_id = "${var.ami}"
  instance_type = "t2.small"

  root_block_device {
    volume_size = "${var.disk_volume_size}"
  }
}

resource "aws_autoscaling_group" "bluegreen_asg" {
  launch_configuration = "${aws_launch_configuration.bluegreen_launchconfig.id}"
  max_size = "2"
  min_size = "1"
}
```

### Expected Behavior
The provider should resize/modify the disk_volume_size attribute on subsequent apply events.

### Actual Behavior
The provider leaves the disk_volume_size attribute the same size it was when created initially.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply` with a specified disk_volume_size
2. `terraform plan/apply` with an updated disk_volume_size
3. No change will register

### References
- https://github.com/hashicorp/terraform/issues/3941

**Comment by stack72** [comment]
Hi @Dombo 

Thanks for reporting this - this was released as a bug fix in Terraform 0.9.6 (https://github.com/hashicorp/terraform/blob/master/CHANGELOG.md#096-may-25-2017)

You will have to upgrade to that version from 0.9.5 to be able to take advantage of it

thanks

Paul

**Comment by p-thurner** [comment]
I'm having the same issue on:

```
terraform --version
Terraform v0.11.8
+ provider.aws v1.38.0
```
